systems/registry/api-gateway/pkg/client/users.go

```diff
@@ -68,20 +79,11 @@ func (r *Users) UpdateUser(userUUID string, user *pbusers.UserAttributes, reques
 	})
 }
 
-func (r *Users) Delete(userUUID string, requesterId string) error {
-	ctx, cancel := r.getContext(requesterId)
-	defer cancel()
-
-	_, err := r.client.Delete(ctx, &pbusers.DeleteRequest{UserUuid: userUUID})
-	return err
-}
-
-func (r *Users) DeactivateUser(userUUID string, requesterId string) error {
+func (r *Users) Delete(userUUID string, requesterId string) (*pbusers.DeleteResponse, error) {
```

Comment:
Can we rename `useruuid` to `userId` for the sake of readability?

Reply:
addressed

Reply:
Yea, your point is valid. but we're calling it `userUUID` in all other endpoints before. So i just followed the conventions. For that readability improvement i think we should address it separately in registry api-gw.